plugins/targetlocker/inmemory/inmemory.go

```diff
@@ -47,18 +47,34 @@ type lock struct {
 	expiresAt time.Time
 }
 
+func validateRequest(req *request) error {
```

Comment:
this method is called in the request broker below - every request needs valid fields